### DISCUSSION TITLE
Is there any Automated Grammar Plugin to Link to MS Office Applications?

### DISCUSSION THREAD

**Discussion by muntasirzisan** [community/community#128379] [Other Feature Feedback, Questions, & Ideas]
Hey Everyone, 

So I've been working as a writer at a firm. Whenever I need to do something quicker or feel that something is repetitive I hop onto VS Code and code something to automate it. But I wanted to create an Automatic Grammar and Writing Tone Checker. But didn't find any. 

Does any one of you know about something similar? If you know then please lemme know it will be super helpful. Peace

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by muntasirzisan** [comment]
Lemme know any !!

**Comment by muntasirzisan** [comment]
It's a delight to have such a generous community out here :)

**Comment by ebndev** [comment]
Hi @muntasirzisan, thanks for being a part of the GitHub Community!

Not sure if this is what you're looking for but [Grammarly](https://www.grammarly.com/microsoft-office) is a popular assistive writing tool. They have an add on for Microsoft office as well. 

If that's not what you meant, please feel free to provide more details 😄
